jsk_fetch_robot/jsk_fetch_startup/launch/fetch_bringup.launch

```diff
@@ -422,6 +423,13 @@
     </group>
   </group>
 
+  <!-- smart device ros -->
+  <group if="$(eval arg('smart_device_port') != '')">
```

Comment:
I'm just commenting because there are no robots running on indigo at JSK right now, so I'm sure it's fine, but I've received comments before that this writing style is deprecated because it is only supported by kinetic and later.

Reply:
Here https://github.com/jsk-ros-pkg/jsk_robot/pull/1720#discussion_r1046673452.

Reply:
But I think it's ok to this PR because that #1720 PR is for jsk_robot_startup.

Reply:
I see, thanks for your comments.